src/OpenTelemetrySDK/OpenTelemetryController.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.Azure.Functions.PowerShellWorker.PowerShell;
+using Microsoft.Azure.Functions.PowerShellWorker.Utility;
+using LogLevel = Microsoft.Azure.WebJobs.Script.Grpc.Messages.RpcLog.Types.Level;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.OpenTelemetrySDK
+{
+    using PowerShell = System.Management.Automation.PowerShell;
+
+    internal class OpenTelemetryController
+    {
+        private readonly ILogger _logger;
+        private PowerShell _pwsh { get; set; }
+
+        private const string StartOpenTelemetryInvocationCmdlet = "Start-OpenTelemetryInvocation";
+        private const string StopOpenTelemetryInvocationCmdlet = "Stop-OpenTelemetryInvocation";
+        private const string GetFunctionsLogHandlerCmdlet = "Get-FunctionsLogHandler";
+
+        private const string OTelEnvironmentVariableName = "OTEL_FUNCTIONS_WORKER_ENABLED";
+
+        private static bool? _isOpenTelemetryCapable;
+        private static bool? _isOpenTelemetryModuleLoaded;
+        private static bool? _isOpenTelmetryEnvironmentEnabled;
+
+        public OpenTelemetryController(ILogger logger, PowerShell pwsh)
+        {
+            _logger = logger;
+            _pwsh = pwsh;
+        }
+
+        public bool isOpenTelemetryCapable()
```

Comment:
Method naming convention: should uppercase the first character

Reply:
Done